tests/kernels/moe/deepep_utils.py

```diff
@@ -79,6 +82,13 @@ def _worker_parallel_launch(
         torch.distributed.destroy_process_group()
 
 
+def find_free_port():
```

Comment:
We can probably move this to utils?

Reply:
Sounds good, moved to `vllm/model_executor/layers/fused_moe/utils.py`